EphraimDev/join-u
# Users can update profile details using the endpoint

#### What does this PR do?
Add the controller and route for update user data
#### Description of Task to be completed?
Have the following endpoint working
POST /api/v1/auth/signup
POST /api/v1/auth/signin
PATCH /api/v1/user/:userId
#### How should this be manually tested?
After cloning the repo, cd into it, set the environment variables and run `npm run start:dev`
* Using postman, test for the above enpoint
* The body of the request for sign up should contain at least `first_name` `last_name` `email` `password`
* The body request for sign in should contain the `email` and `password`
* Set the `token` in the header as `authorization` to test the update profile endpoint
* `bearer ` to the token
#### What are the relevant pivotal tracker stories?
#167689406
